src/index.js

```diff
@@ -4,8 +4,8 @@ import cx from 'classnames';
 import debounce from './lib/debounce';
 
 class OnVisible extends Component {
-    constructor() {
-        super(...arguments);
+    constructor(props) {
+        super(props);
```

Comment:
I'm sure I saw somewhere `super(props, context)`, which was what I was thinking of. Maybe in the source.. I'll have to look.